evap/evaluation/tests/test_views.py

```diff
@@ -246,3 +248,43 @@ def test_notebook(self):
 
         user.refresh_from_db()
         self.assertEqual(user.notes, self.note)
+
+
+class TestResetEvaluation(WebTestStaffMode):
+
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
A bit annoying, but pep-484 defaults to "Any" instead of "None" for methods that return nothing
```suggestion
    def setUpTestData(cls) -> None:
```